client/my-sites/media-library/test/list.jsx

```diff
@@ -79,6 +75,10 @@ describe( 'MediaLibraryList item selection', () => {
 			mediaList = wrapper.find( MediaList ).instance();
 		} );
 
+		afterEach( () => {
```

Comment:
huh. why did these fail?

Reply:
It seems `wrapper.unmount()` does actually unmount the component instance but not remove the reference to `wrapper` itself. In the previous condition we were checking if `wrapper` exists (it does, in any test) and then call `wrapper.unmount()` regardless of its mounted state. That produced the error.

As we only use `mount()` in two tests (there are several more) I moved the `wrapper.unmount()` call to these.